src/NuGet.Core/NuGet.CommandLine.XPlat/ReportRenderers/Models/ListPackageOutputContentV1.cs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+
+namespace NuGet.CommandLine.XPlat
+{
+    /// <summary>
+    /// List package json output version 1
+    /// </summary>
+    internal class ListPackageOutputContentV1
```

Comment:
Is this supposed to be just `ListPackageOutputContent`? 

Right now we have the version in the name and in the version. 
Wondering if we need both?

Maybe I'm over thinking it :)

Reply:
I'll add that this does not match my idea of what an MVC pattern is. Not that there's a right or wrong answer, it's just a high level idea. But to me a model is the internal representation of the program's data. it doesn't matter if it's outputting text or json, or v1 or v2 output, there's one internal memory representation of the data.  Then there can be a text renderer that writes it to console in a specific way. There can be a JsonV1 renderer that formats it in a certain way and a JsonV2 renderer that outputs it in a different way.  But versioning models isn't how I think about designing MVC, because it means that the "service" to get data from your data sources can't be reused for v1 and v2.

edit: reading the code a bit more, if this data structure continues to be useful, since it's only used for json formatting, in my opinion it belongs in the `JsonRenderer` directory, not the shared models.  Having said that, I'm a big fan of per-feature, or per-controller folders that contain models and views in the same directory, rather than a directory full of every view and a different directory with every model. To me, how it's organized should be optimized to make it quick and easy to find code that you're likely to work on at the same time. But I know that organization that other code organization pattern is used (unfortunately by the ASP.NET MVC new project template).

Reply:
You're right, I removed versioning from here. Technically I can remove this file altogether since I merged writer into renderer file, it's not really necessary.